### PR TITLE
VACMS-11011: Remove Vagovadmin theme.

### PR DESCRIPTION
**Pull request by ndouglas**
## Description

Relates to #11011.  This is going to have a lengthy QA process, most likely.

### CODE REVIEW

File: docroot/themes/custom/vagovadmin/yarn.lock [hunk 1]
```diff
@@ -2874,7 +2874,7 @@ isarray@1.0.0, isarray@~1.0.0:
 isarray@2.0.1:
   version "2.0.1"
   resolved "https://registry.yarnpkg.com/isarray/-/isarray-2.0.1.tgz#a37d94ed9cda2d59865c9f76fe596ee1f338741e"
-  integrity sha512-c2cu3UxbI+b6kR3fy0nRnAhodsvR9dx7U5+znCOzdj6IfP3upFURTr0Xl5BlQZNKZjEtxrmVyfSdeE3O57smoQ==
```
Comment (swirtSJW) [inline docroot/themes/custom/vagovadmin/yarn.lock:2877]: I think I am a little confused as to why this file it being altered and not just removed.

**Review by swirtSJW — COMMENTED** [review]
I was kind of expecting to see all of themes/custom/vagovadmin get removed in this PR.   Does this have to be done in two steps?
Other than that the code changes make sense and look very thorough.

Have not poked around to make sure the site is looking right yet.

**Comment by ndouglas** [comment]
>I was kind of expecting to see all of themes/custom/vagovadmin get removed in this PR. Does this have to be done in two steps?

Yep, needs two steps unfortunately.  

>Other than that the code changes make sense and look very thorough.

Yeah, I disabled `vagovadmin` first just to see what would happen, and a lot of things broke -- which was when I realized we had a whole lot of stuff that I think was never properly updated.  

I think I'll probably need to go through each of the blocks, scripts, files, etc, figure out where exactly it's used, then confirm that it's functioning as intended.  Maybe open a PR based off this one that removes `vagovadmin` from the filesystem and actually use that one for the manual testing.

That all sounds suspiciously like work, so I haven't done it yet 😉

File: docroot/themes/custom/vagovadmin/yarn.lock (hunk 1 above)
Comment (ndouglas) [reply docroot/themes/custom/vagovadmin/yarn.lock:2877]: Yeah, that's weird.  I wonder if I was using the wrong version of `yarn` or something.  Honestly can't remember exactly why I was tinkering with the `vagovadmin` installation anyway 🤔 

I might need to go through and sort of retrace my steps when I return to this, now that I have a better picture of what needs to be done.  Maybe redo it from scratch 🤔   Probably the wise thing to do would be to carry out the changes that didn't get fully implemented, test those on an individual basis, and then ideally zapping the old theme should be trivial.

**Pull request closed without merging**